modules/extensions/autoscaler.tf

```diff
@@ -148,3 +157,15 @@ resource "null_resource" "cluster_autoscaler" {
     inline = ["kubectl apply -f ${local.cluster_autoscaler_manifest_path}"]
   }
 }
+
+resource "null_resource" "local_cluster_autoscaler" {
+  count = local.local_cluster_autoscaler_enabled ? 1 : 0
+
+  triggers = {
+    manifest_md5 = try(md5(local.cluster_autoscaler_manifest), null)
+  }
+
+  provisioner "local-exec" {
+    command = "cat ${local.cluster_autoscaler_manifest} | kubectl apply --dry-run='client' -f -"
```

Comment:
cat-ing that local variable into kubectl apply doesn't seem to work. I get an error like the following:

```
│       serviceAccountName: cluster-autoscaler-oci-oke-cluster-autoscaler                                      
│       tolerations:                                                                                                           
│         []                                                                                                                   
│  | kubectl apply --dry-run='client' -f -': exit status 2. Output: : -: not found                             
│ /bin/sh: 81: -: not found                                    
│ /bin/sh: 82: -: not found                                                                                                    
│ /bin/sh: 83: -: not found                                    
│ /bin/sh: 84: -: not found 
```

Maybe there are newlines in the variable that is messing it up?

Options:
* Use the helm provider that is already in `versions.tf` to just apply the helm chart.
* Use the `local_file` resource to actually save the manifest yaml, which would have a similar effect to what the remote-exec is doing.